audb/core/load.py

```diff
@@ -1073,7 +1066,7 @@ def load_media(
                     db_root,
                     name,
                     version,
-                    cached_versions,
+                    None,
```

Comment:
I think I would prefer to define `cached_versions = None` before as it is then easier to understand what is passed on instead of just `None`

Reply:
Ok, but in that case it does also make sense to assign the result of the function back to `cached_version` as it was before.

Reply:
Why?

Reply:
because if we ever change the code for some reason and continue using `cached_version` it's still `None` otherwise.

Reply:
OK, let's stay with your current solution of directly providing `None` as argument.